aspnetcore/fundamentals/logging/index/samples/3.x/CustomLogger/ColoredConsoleLogger/ColoredConsoleLogger.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace CustomLogger.ColoredConsoleLogger
+{
+    #region snippet
+    public class ColoredConsoleLogger : ILogger
+    {
+        private readonly string _name;
+        private readonly ColoredConsoleLoggerConfiguration _config;
+
+        public ColoredConsoleLogger(string name, ColoredConsoleLoggerConfiguration config)
+        {
+            _name = name;
+            _config = config;
+        }
+
+        public IDisposable BeginScope<TState>(TState state)
+        {
+            return null;
+        }
+
+        public bool IsEnabled(LogLevel logLevel)
+        {
+            return logLevel == _config.LogLevel;
```

Comment:
Might want to use `>=` for the check instead

Reply:
Not, if you want to set the color for a specific log level

Reply:
@JuergenGutsch  reverted to your original code